### ISSUE TITLE
Support checkboxes with live update

### ISSUE THREAD

**Issue by nebulade** [nebulade/meemo#59]
Currently checkboxes are only supported as static markdown elements. We should add the ability to toggle them directly and permanently.

**Comment by Frankstar** [comment]
this would improve meemo alot.

**Comment by nebulade** [comment]
Agree, I gave it a try some time ago, but it was going nowhere so far. Hopefully I get around to is eventually.

**Comment by Frankstar** [comment]
https://github.com/Laverna/laverna has a nice way of doing it.
Saw the same approach (exactly the same!) on 2 other products - but cant remember them.

**Comment by Frankstar** [comment]
i heard you will implement this in 2019, is it true???
:D

**Comment by nebulade** [comment]
Right, I don't really have much time for that though. I had a couple of attempts for this already, but the markdown rendering engine isn't all too suitable for that as far as I remember :-/
